src/main/java/tutorease/address/logic/commands/AddLessonCommand.java

```diff
@@ -59,7 +59,7 @@ public AddLessonCommand(StudentId studentId, StartDateTime startDateTime, Locati
     @Override
     public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
-        ObservableList<Person> personList = model.getTutorEase().getPersonList();
+        ObservableList<Person> personList = model.getFilteredPersonList();
```

Comment:
This will work well if we intend to create lessons from a filtered list of contacts, after using the find contact command.